addons/sourcemod/configs/szf/weapons.cfg

```diff
@@ -656,7 +661,7 @@
 		"589"	//Eureka Effect
 		{
 			"text"		"Melee_EurekaEffect"
-			"attrib"	"286 ; 1.333 ; 352 ; 0.0"
+			"attrib"	"113 ; 12.5 ; 352 ; 0.0"
```

Comment:
12.5 metal? I'd rather just have a flat number like 10.

Reply:
It's affected by the 20% metal reduction

Reply:
Fair, even though it doesn't make sense that the metal regen is affected by attribute 732